build_all/windows/build.cmd

```diff
@@ -220,7 +220,7 @@ if %MAKE_NUGET_PKG% == yes (
         if !ERRORLEVEL! neq 0 exit /b !ERRORLEVEL!
 
         if %build-platform% neq arm (
-            ctest -T test --no-compress-output -C "debug" -V -j 8
+            ctest -T test --no-compress-output -C "debug" -V -j 10
```

Comment:
>10 [](start = 64, length = 2)

Nit: Recommend we bump this to 16 so that we have more headroom, and won't hurt in meantime.